macros/CSSInfo.ejs

```diff
@@ -11,21 +11,7 @@ let name = $0 || (slug ? slug.split("/").pop().toLowerCase() :
     "preview-wiki-content");
 let atRule = $1;
 
-let propertiesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/properties.json";
-let atRulesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/at-rules.json";
-let selectorsUrl = "https://raw.githubusercontent.com/mdn/data/master/css/selectors.json";
-let typesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/types.json";
-let syntaxesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/syntaxes.json";
-let unitsUrl = "https://raw.githubusercontent.com/mdn/data/master/css/units.json";
-let localStringsUrl = "https://raw.githubusercontent.com/mdn/data/master/l10n/css.json";
```

Comment:
In my local dev environment, I tried using this macro and it failed because `localStringsUrl` is still used below:
```
let localStrings = await MDN.fetchJSONResource(localStringsUrl);
```